crest/Assets/Crest/Crest/Scripts/Helpers/TextureArrayHelpers.cs

```diff
@@ -80,14 +92,17 @@ static void InitStatics()
             // Init here from 2019.3 onwards
             sp_LD_TexArray_Target = Shader.PropertyToID("_LD_TexArray_Target");
 
-            if (BlackTextureArray == null)
+            if (_blackTextureArray == null)
             {
-                BlackTextureArray = CreateTexture2DArray(Texture2D.blackTexture);
-                BlackTextureArray.name = "Black Texture2DArray";
+                _blackTextureArray = CreateTexture2DArray(Texture2D.blackTexture);
```

Comment:
We shuld probably just access the public getter or collapse this code into a function that is called from two places.

Reply:
good call, factored it out, thx